Src/VimCore/CommandUtil.fs

```diff
@@ -102,6 +101,9 @@ type internal CommandUtil
 
     let mutable _inRepeatLastChange = false
 
+    /// The last mouse down position before it was adjusted for virtual edit

```

Comment:
This will be kept in sync for all of the moves that we do. But what about mouse moves that happen externally? For instance another extension doing the move or the caret moving due to an insert? Does it need to be cleared for those?

Reply:
Our only requirement is that a mouse drag (which can only occur when the mouse button is down) is preceded by a mouse click, and the mouse device assures us of that. The time to clear it is on mouse up but that contract just mentioned makes that unnecessary.

Other extensions or insert mode can only move the caret, not the mouse and this variable is unrelated to the caret.

If not clearing it just feels wrong, I can add `<LeftUp>` bindings for all modes. In fact, I'll do that because it is remotely conceivable that the mouse up event occurs at a position for which a drag event was not generated and the drag is only to provide feedback between down and up.